### ISSUE TITLE
mesa fails to build

### ISSUE THREAD

**Issue by JaroslavHron** [hashdist/hashstack#783]
The requiered version is not at the give URL anymore....

```
Downloading ftp://ftp.freedesktop.org/pub/mesa/10.2.4/MesaLib-10.2.4.tar.bz2...
[ERROR] urllib failed to download (reason: ftp error: 550 Failed to change directory.): ftp://ftp.freedesktop.org/pub/mesa/10.2.4/MesaLib-10.2.4.tar.bz2
[CRITICAL] You may wish to check your Internet connection or the remote server

```

fixed by
```
sources:
- key: tar.bz2:a2rdiese5oc4fa7vt5ybmhqg33iqn6bv
  url: ftp://ftp.freedesktop.org/pub/mesa/older-versions/10.x/10.2.4/MesaLib-10.2.4.tar.bz2
```

**Comment by certik** [comment]
Cool, thanks! Can you send a pull request please? Then we can merge it.

**Comment by certik** [comment]
Fixed by #785.
